elasticsearch/elasticsearch.yml

```diff
@@ -8,6 +8,9 @@ http.cors:
   allow-methods: OPTIONS, HEAD, GET, POST, PUT, DELETE
   allow-headers: "X-Requested-With, Content-Type, Content-Length, if-modified-since, Authorization"
 
+http.max_content_length: 500mb
```

Comment:
Why is this necessary? Is this trying to index a single file thats huge?

https://www.elastic.co/guide/en/elasticsearch/reference/current/modules-network.html suggests to really not do that.

If sending a smaller file does `indexing_pressure.memory.limit` need to be there?

Reply:
Thanks, yes, it was a bit sloppy to increase limits instead of splitting the files to achieve smaller bulk indexing request. I've changed the indexing bash script and split the files now and have removed the limits.